### ISSUE TITLE
disableOnClickOutside doesn't seem to do anything

### ISSUE THREAD

**Issue by poohsen** [YouCanBookMe/react-datetime#402]
### I'm Submitting a ...

```
[x] Bug report
[ ] Feature request
[ ] Support request
```

### Steps to Reproduce
render the following:
```
<Datetime
   disableOnClickOutside={true}
 />
```
### Expected Results
When clicking outside of the datepicker I expect nothing to happen, i.e. the widget to stay open.

### Actual Results
The widget closes. The behaviour is the same when `disableOnClickOutside={true}` is changed to `disableOnClickOutside={false}`

### Other Information (e.g. stacktraces, related issues, suggestions how to fix)
reproduced in react-datetime version: 2.8.10 and 2.9
I'm using `create-react-app` and running `npm start`. I tried the latest Chrome and FF.

**Comment by simeg** [comment]
I can't reproduce this. Could you provide a codepen that displays this error? You can use [this one](https://codepen.io/simeg/pen/mEmQmP) which has the component set up.

**Comment by simeg** [comment]
Fixed in #405.
